src/DrHouse/HealthData.cs

```diff
@@ -14,12 +16,16 @@ public HealthData(string appName)
             DependenciesStatus = new List<HealthData>();
         }
 
+        [DataMember]
         public string Name { get; private set; }
 
+        [DataMember]
         public string Type { get; set; }
 
+        [DataMember]
         public string ErrorMessage { get; set; }
 
+        [DataMember]
```

Comment:
Please move this attribute to the property IsOk, otherwise the chain validation will not be triggered.